RubberduckTests/Mocks/MockVbeBuilder.cs

```diff
@@ -53,12 +54,21 @@ public class MockVbeBuilder
             ["ADOR"] = LibraryPathAdoRecordset
         };
 
+        private static readonly Dictionary<string, Action<MockProjectBuilder>> addReferenceActions = new Dictionary<string, Action<MockProjectBuilder>>
+        {
```

Comment:
Can weake this a dictionary of `Func<MockProjectBuilder, MockProjectBuilder>` please to keep the fluid nature of the builder methods?

Reply:
On a different subject, I'm wondering if since we are cleaning up how references are handled why not use a `enum` or a static class with static members instead of `string` to key the references? Using a static class would make it easy for us to write the necessary data for any new reference we wish to add and have a compiler-validated type on all references?

Reply:
I agree, what we really want here is an enum of libraries together with an overload of `AddReference` taking a `params` array of enum elements.

Reply:
I like the idea of moving away from string keys but I think doing so should be the focus of a different PR.  That type of modification will introduce changes to many test files other than inspection tests.